app/src/test/java/ch/epfl/sdp/database/firebase/api/CommonMockDatabaseAPI.java

```diff
@@ -11,8 +11,8 @@
 import ch.epfl.sdp.database.utils.OnValueReadyCallback;
 
 public class CommonMockDatabaseAPI implements CommonDatabaseAPI {
-    private Map<String, UserForFirebase> userForFirebaseMap = new HashMap<>();
-    private Map<String, PlayerForFirebase> playerForFirebaseMap = new HashMap<>();
+    public Map<String, UserForFirebase> userForFirebaseMap = new HashMap<>();
```

Comment:
You need to access this somewhere?

Reply:
Yes, in line 55.

Reply:
I mean, since it's in the same file, you don't need it to be public no?